src/test/java/com/google/ehub/servlets/ItemSubmissionServletTest.java

```diff
@@ -0,0 +1,124 @@
+package com.google.ehub.servlets;
+
+import static org.mockito.Mockito.when;
+
+import com.google.appengine.tools.development.testing.LocalDatastoreServiceTestConfig;
+import com.google.appengine.tools.development.testing.LocalServiceTestHelper;
+import com.google.ehub.data.EntertainmentItemDatastore;
+import java.io.IOException;
+import java.util.Arrays;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import org.junit.After;
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.junit.runners.JUnit4;
+import org.mockito.Mock;
+import org.mockito.MockitoAnnotations;
+
+@RunWith(JUnit4.class)
+public class ItemSubmissionServletTest {
+  private final ItemSubmissionServlet servlet = new ItemSubmissionServlet();
+  private final EntertainmentItemDatastore entertainmentItemDatastore =
+      EntertainmentItemDatastore.getInstance();
+  private final LocalServiceTestHelper helper =
+      new LocalServiceTestHelper(new LocalDatastoreServiceTestConfig());
+
+  private static final String TITLE_PARAMETER_KEY = "Title";
+  private static final String DESCRIPTION_PARAMETER_KEY = "Plot";
+  private static final String IMAGE_URL_PARAMETER_KEY = "Poster";
+
+  private static final int MAX_TITLE_CHARS = 150;
+  private static final int MAX_DESCRIPTION_CHARS = 500;
+
+  @Mock HttpServletRequest request;
+
```

Comment:
nit: you can remove the extra blank line

Reply:
I removed the extra line in last commit.